make.js

```diff
@@ -401,14 +401,15 @@ function categorizeCalls(apis) {
 function setPrerequisiteCalls(apis) {
     for (var i = 0; i < apis.length; i++) {
         var api = apis[i];
-        prereqs = [];
+        prerequisiteApis[api.name] = [];

```

Comment:
>prerequisiteApis[api.name] [](http://example.com/codeflow?start=8&length=26)

What is the format of this data structure?  Asking because it will most likely be impacted by the recategorization work I'm doing now.

Reply:
I tried to keep it as close to categorizedApis { "apiName": loginCalls[], otherCalls[] } as possible, so it's a dictionary in the form { "apiName": prereqCalls[] }